polly/test/Support/dumpmodule.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
```

Comment:
No they haven't. Drop this line and and the spurious changes to the test.

Reply:
fixed